src/foam/u2/tag/Select.js

```diff
@@ -20,23 +20,26 @@ foam.CLASS({
   name: 'Select',
   extends: 'foam.u2.View',
 
+  implements: [
+    'foam.u2.ChoiceViewTrait'
```

Comment:
I was following the lead of FOAM 1 here. On the other hand, `ChoiceViewTrait` could have been a superclass rather than a trait in FOAM 1 as well.

Should I move it to be a parent class instead?